src/outputProcessor/BaseOutputProcessor.d.ts

```diff
@@ -1,29 +1,34 @@
 /**
- * Gives the functions and variables available for use when specifying a custom output processor.
- * i.e. Allows use of this.writeSpecChunk without ts warning/error
- * 
- * Example usage:
- * 
- * import { AllMessages } from 'cypress-terminal-report/src/installLogsPrinter'
- * import { BaseOutputProcessor } from 'cypress-terminal-report/src/outputProcessor/BaseOutputProcessor'
- * 
- * ...
- * 
- * outputTarget: {
- *     'custom|cts':  function outputProcessor(this: BaseOutputProcessor, messages: AllMessages){ <custom output processor function> }                
```

Comment:
Moved this example to `CustomOutputProcessor` JSdocs, because  `CustomOutputProcessor`  is what is actually used here (as `this`), not `BaseOutputProcessor`